config/selectionCfg_ETau_UL16.cfg

```diff
@@ -0,0 +1,1279 @@
+#bTagweightM = bTagSFMUp:bTagweightM_up, bTagSFMDown:bTagweightM_down
+#bTagweightL = bTagSFLUp:bTagweightL_up, bTagSFLDown:bTagweightL_down
+#TTtopPtreweight = topUp:TTtopPtreweight_up , topDown:TTtopPtreweight_down
+
+# ROOT-like definition of the binning
+[histos]
+nbjetscand              = 20, 0, 20
+njets                   = 10, 0, 10
+njets20                 = 10, 0, 10
+njets50                 = 10, 0, 10
+npv                     = 50, 0, 50
+npu                     = 25, 0, 50
+
+dau1_pt                 = 20, 0, 200
+dau1_eta                = 21, -2.1, 2.1
+dau1_phi                = 25, -3.14159, 3.14159
+dau1_iso                = 24, 0.8, 1.
+dau1_dxy                = 100, -0.01, 0.01
+dau1_dz                 = 100, -0.1, 0.1
+dau2_pt                 = 20, 0, 200
+dau2_iso                = 24, 0.8, 1.
+dau2_eta                = 21, -2.1, 2.1
+dau2_phi                = 25, -3.14159, 3.14159
+dau2_dxy                = 100, -0.01, 0.01
+dau2_dz                 = 100, -0.1, 0.1
+
+dau1_deepTauVsMu		= 10, -0.5, 9.5
+dau2_deepTauVsMu		= 10, -0.5, 9.5
+
+dau1_deepTauVsJet		= 10, -0.5, 9.5
+dau2_deepTauVsJet		= 10, -0.5, 9.5
+
+dau1_deepTauVsEle		= 10, -0.5, 9.5
+dau2_deepTauVsEle		= 10, -0.5, 9.5
+
+met_et					= 35, 0, 210
+met_phi					= 25, -3.2, 3.2
+metnomu_et				= 35, 0, 210
+metnomu_phi				= 25, -0.1, 6.3
+met_er_et				= 30, 0, 600
+met_er_phi				= 25, -3.14159, 3.14159
+
+HT20Full                = 25, 100, 1000
+
+bjet1_pt                = 30, 20, 200
+bjet1_eta               = 25, -2.5, 2.5
+bjet1_phi               = 25, -3.14159, 3.14159
+bjet1_bID_deepFlavor    = 15, 0, 1
+bjet2_pt                = 30, 20, 200
+bjet2_eta               = 25, -2.5, 2.5
+bjet2_phi               = 25, -3.14159, 3.14159
+bjet2_bID_deepFlavor    = 15, 0, 1
+bjet1_JER               = 35, 0, 400
+bjet2_JER               = 35, 0, 400
+
+btau_deltaRmin          = 25, 0, 5
+btau_deltaRmax          = 25, 0, 5
+
+bH_mass                 = 30, 0, 300
+bH_pt                   = 30, 0, 600
+dib_deltaR              = 25, 0, 5
+dib_deltaPhi            = 15, 0., 3.14159
+dib_deltaEta            = 20, 0, 10
+
+tauH_mass               = 30, 0, 300
+tauH_pt                 = 30, 0, 600
+ditau_deltaR            = 25, 0, 5
+ditau_deltaPhi          = 15, 0., 3.14159
+ditau_deltaEta          = 20, 0, 10
+
+HH_mass                 = 50, 0, 3000
+HH_pt                   = 30, 0, 300
+HHKin_mass              = 50, 0, 3000
+HH_deltaPhi             = 15, 0., 3.14159
+HHsvfit_pt              = 30, 0, 300
+HHsvfit_deltaPhi        = 25, 0, 3.14159
+HH_deltaR               = 35, 0, 7
+
+fatjet_softdropMass					= 50, 0, 300
+fatjet_pt							= 15, 250, 700
+fatjet_eta							=  7, -2.5, 2.5 
+fatjet_phi							=  7, -3.14159, 3.14159
+fatjet_particleNetMDJetTags_score	= 15, 0.0, 1.05
+fatjet_particleNetMDJetTags_mass	= 15, 0, 250
+
```

Comment:
This list seems the most complete to me so I would use this one to check the others.